release-notes/docs/3.7.0.md

```diff
@@ -0,0 +1,202 @@
+
+## Configurable hierarchies
+
+It is now possible to configure contact hierarchies.
+This allows for better tailored solutions for new projects and opens up new configuration 
+possibilities: for example having any number of place levels, multiple person types, or unbalanced place trees. 
+The previously hardcoded contact hierarchy has now become the default hierarchy and 
+existing projects will continue using it. For more information, please read the
+[configuration information](https://github.com/medic/medic-docs/blob/master/configuration/app-settings.md#configuring-the-contact-hierarchy).
+
+## Includes a new default configuration
+
+Version 3.7.0 includes a complete default configuration that is enabled automatically 
+on fresh application installations. The configuration is designed as a reference application 
+for antenatal care. It provides a foundation for forms, data fields, 
+and analytics to model an ideal way of structuring and organizing 
+a workflow, its configuration code, and testing framework.
+
+## Optimised form HTML generation
+
+Generating the HTML for XForms is moved from being executed on every device to 
+being executed once on the server. 
+This reduces the time to initially load forms, along with reducing the memory 
+footprint of the app by not caching every form's `html` and by not storing the additional files
+required for generating it. 
+
+## Improved purging now done server-side
+
+Marking documents as no longer useful so that they don't take up mobile device resources 
+is now handled server-side.
+This enables devices to not download unnecessary documents during an initial replication 
+and makes the actual "purging" process faster. 
+For more information about changes in purging, please refer to the [purging documentation](https://github.com/medic/medic-docs/blob/master/configuration/purging-serverside.md).
+
+## Warn when replicating too many docs
+
+A warning is now displayed when a user would download more than the recommended maximum of 
+10.000 documents. This warning is shown before starting initial replication in an attempt 
+to reduce the incidence of misconfigured users.
+
+![Warn when replicating too many docs](images/3.7.0-replication-warning.png)
+
+## Performance
+
+Users can expect faster replication, faster application load time and faster form load time:
+
+| Action                  | Tecno-J5 on 3.7 | Tecno-J5 on 3.6 | Tecno-WX3 on 3.7 | Tecno-WX3 on 3.6 |
+|-------------------------|-----------------|-----------------|------------------|------------------|
+| Initial load (*)        | 0:02:29         | 0:04:18         | 0:02:45          | 0:05:01          |
+| Loading Contacts Tab    | 0:00:02         | 0:00:02         | 0:00:02          | 0:00:03          |
+| Loading reports tab     | 0:00:20         | 0:00:32         | 0:00:25          | 0:00:44          |
+| Process a specific form | 0:00:19         | 0:00:23         | 0:00:21          | 0:00:34          |
```

Comment:
It would be cool to see this as a bar graph showing how significant the differences are